src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java

```diff
@@ -1213,7 +1213,13 @@ public void consume(int x, int y, int id) {
 							return true;
 						} else if (keyPressed == NotEnoughUpdates.INSTANCE.config.misc.openAHKeybind) {
 							String cleanName = Utils.cleanColour(item.get("displayname").getAsString()).replace("[Lvl {LVL}]", "").trim();
-							if (NotEnoughUpdates.INSTANCE.manager.auctionManager.getBazaarInfo(internalname.get()) == null) {
+
+							if (item.get("itemid").getAsString().equals("minecraft:enchanted_book")) {
```

Comment:
Checking just for itemid will break for items that render as an enchanted book, but are *not* an enchanted book. For example the experience artifact.